cmd/cert-checker/main.go

```diff
@@ -51,8 +51,8 @@ type report struct {
 func (r *report) save(directory string) error {
 	filename := path.Join(directory, fmt.Sprintf(
 		"%s-%s-report.json",
-		r.begin.Format(filenameLayout),
 		r.end.Format(filenameLayout),
+		r.begin.Format(filenameLayout),
```

Comment:
So begin is *where* the run started scanning from, not when, and end is the end of the period scanned, so it previously went `(today)-(today-90 days).json` which was somewhat confusing, instead of switching the ordering of the print args though it might just make sense to switch the assignments of `begin`/`end` so that they refer to the earliest date scanned to the last date scanned respectively.